### ISSUE TITLE
Request: hide header and show footer

### ISSUE THREAD

**Issue by ASNNetworks** [KTibow/lovelace-light-soft-ui-theme#9]
I dug thourgh the theme.yaml, but I couldn’t figure out if this code was present. Basically, I want the header to be compact and as the footer. So nothing as header and only a compact footer. 

Also, the tab selection ‘thingy’ needs to be at the top and the option to color active and inactive tabs. No chevrons either and bottom padding. Is this possible?

Picture for reference:

![ACB5BF80-6B7F-497F-B18D-2065108D2946](https://user-images.githubusercontent.com/54230480/95601494-d1999a00-0a53-11eb-8cd8-40de1df6f2d4.jpeg)

**Comment by KTibow** [comment]
It's possible. Putting this in my saved notifications.

**Comment by KTibow** [comment]
Config is a problem. Some people want it at the top, some at the bottom. Any config ideas? You could try to install some JS that adds on that CSS.

**Comment by ASNNetworks** [comment]
> Config is a problem. Some people want it at the top, some at the bottom. Any config ideas? You could try to install some JS that adds on that CSS.

Honestly, for me just the code is enought since I have my own theme.yaml that I maintain :) Perhaps you can add the # in front in this theme so people can decide which one they want.

**Comment by KTibow** [comment]
Yahoo, release on the way! Close when ready.

**Comment by KTibow** [comment]
Refresh HACS by rebooting, or clicking HACS > Frontend > scroll to my theme > 3 dots > Update information > wait > click update

**Comment by ASNNetworks** [comment]
> Refresh HACS by rebooting, or clicking HACS > Frontend > scroll to my theme > 3 dots > Update information > wait > click update

Awesome! Could you perhaps point me to the piece of code, so I can use that in other theme.yaml files?

**Comment by KTibow** [comment]
It's here:
https://github.com/KTibow/lovelace-light-soft-ui-theme/blob/c100d1627ae8605e102ec72e9fb2ea95921347dd/theme.yaml#L261-L374

**Comment by ASNNetworks** [comment]
> It's here:
> https://github.com/KTibow/lovelace-light-soft-ui-theme/blob/c100d1627ae8605e102ec72e9fb2ea95921347dd/theme.yaml#L261-L374

Thanks, I can't seem to get it to work. I have:

```
  card-mod-root-yaml: |
    paper-tabs:
      $: |
        .not-visible {
          display: none;
        }
        selectionBar {
          bottom: unset !important;
        }
    "ha-app-layout":
      $: |
        contentContainer {
          padding-top: 0px !important;
        }
    .: |
      app-toolbar {
        height: 0;
      } 
```
Did I mess it up with uncommenting? Tried multiple times, but I can't seem to get it to work. Btw I also have more-info styling. so the complete card mod setup in my theme.yaml is:

```
  card-mod-theme: night-footer
  card-mod-more-info-yaml: |
    $: |
      .mdc-dialog {
        -webkit-backdrop-filter: blur(17px);
        backdrop-filter: blur(17px);
        background: rgba(0,0,0,0.25);
      }
      .mdc-dialog .mdc-dialog__container .mdc-dialog__surface {
        background: none !important;
        box-shadow: none;
        border-radius: 0px;
      }
    ha-header-bar:
      $: |
        .mdc-top-app-bar {
          background: none !important;
        }
  card-mod-root-yaml: |
    paper-tabs:
      $: |
        .not-visible {
          display: none;
        }
        selectionBar {
          bottom: unset !important;
        }
    "ha-app-layout":
      $: |
        contentContainer {
          padding-top: 0px !important; /*Change this to 0px if you want the header on the bottom*/
        }
    .: |
      app-toolbar {
        height: 0;
      }             
```

**Comment by KTibow** [comment]
Select all of it, not just the top part.

**Comment by KTibow** [comment]
Mind if I close?

**Comment by ASNNetworks** [comment]
> Mind if I close?

I can't get it to work, but I suspect I have something else that prevents it. I can't even get the simple code to remove top part of header to work (https://github.com/thomasloven/lovelace-card-mod/wiki/Theme-cookbook).

You can close this, since I probably have some other issues why this isn't working.

**Comment by KTibow** [comment]
Probably card-mod-theme, make sure it's correct, also make sure card-mod is installed, also try refreshing the page.

**Comment by ASNNetworks** [comment]
> Probably card-mod-theme, make sure it's correct, also make sure card-mod is installed, also try refreshing the page.

Yeah, I already use card mod for quite some time. I already have it setup with more-info in my theme yaml. Refreshed lik 30x on different browsers. I guess I'll keep investigating. Just to confirm: this does work on HA 115 right, or do those codes only work on HA 116?

**Comment by KTibow** [comment]
Should be cross-compatible, just updated and seems to still work.

**Comment by ASNNetworks** [comment]
> Should be cross-compatible, just updated and seems to still work.

I got it to work! I didn't realize I had to uninstall CH as well (I was trying to recreate it through card mod first before removing CH). I now have the footer working :)

Two things: is it possible to add padding to the footer (so it's a little higher)? Also, somehow I can't seem to delete the chevrons. 

This is how it now looks:
![IMG_0877(1)](https://user-images.githubusercontent.com/54230480/95618632-b471c500-0a6d-11eb-9e0c-1c591ddab81c.PNG)

I need the icons to go a little higher with a bottom padding of 30px and more spread out and lose the chevrons. Is that possible?

I wanted to re-open this issue, but not sure what is ideal.
